### PR TITLE
[DOC] update readme

### PR DESCRIPTION
**Pull request by Remi-Gau**
quick clean up of the README

left some things commented as I do not know if they apply (for example: passing a config to the bids validator)

rendered version: https://github.com/Remi-Gau/python-validator/blob/readme/README.md

### CODE REVIEW

**Comment by Remi-Gau** [comment]
more possible changes:
- remove contributors as they correspond to those in the javascript validator
- remove zenodo.json for the same reason

**Comment by Remi-Gau** [comment]
is this failing because I opened the PR from a fork?

https://github.com/bids-standard/python-validator/actions/runs/10384933325/job/28752987383?pr=7

**Comment by effigies** [comment]
Seems plausible. I can try disabling that on forks.

File: README.md [hunk 1]
```diff
@@ -1,80 +1,72 @@
...
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3688707.svg)](https://doi.org/10.5281/zenodo.3688707)
+[![PyPI version](https://badge.fury.io/py/bids-validator.svg)](https://badge.fury.io/py/bids-validator)
+[![Conda version](https://img.shields.io/conda/vn/conda-forge/bids-validator)](https://anaconda.org/conda-forge/bids-validator)
+
+# Python BIDS-Validator
+
+This is a library of helper functions written in Python,
+for use with BIDS compliant applications written in this language.
+
+The main function determines if a file extension is compliant with the BIDS specification.
```
Comment (effigies) [inline README.md:12]:
```suggestion
The main function determines if a file path is compliant with the BIDS specification.
```

File: README.md [hunk 2]
```diff
@@ -1,80 +1,72 @@
...
+
+## Installation
+
+To install with pip:
+
+```
+pip install -U bids_validator
```
Comment (effigies) [inline README.md:19]:
```suggestion
python -m pip install bids_validator
```

File: README.md [hunk 3]
```diff
@@ -1,80 +1,72 @@
...
+## Acknowledgments
+
+Many contributions to the `bids-validator` were done by members of the
+BIDS community. See the
+[list of contributors](https://bids-specification.readthedocs.io/en/stable/99-appendices/01-contributors.html).
+
+A large part of the development of `bids-validator` is currently done by
+[Squishymedia](https://squishymedia.com/), who are in turn financed through
+different grants offered for the general development of BIDS. See the list
+below.
```
Comment (effigies) [inline README.md:61]:
```suggestion
A large part of the initial development of `bids-validator` was done by
[Squishymedia](https://squishymedia.com/), who are in turn financed through
different grants offered for the general development of BIDS. See the list
below.
```

**Review by effigies — COMMENTED** [review]
Initial comments

**Comment by Remi-Gau** [comment]
will merge and we can revert or iterate on this

**Pull request merged**